recipes/chaiscript/all/conanfile.py

```diff
@@ -21,60 +29,78 @@ class ChaiScriptConan(ConanFile):
                        "header_only": True}
 
     @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    def _min_cppstd(self):
+        return 17
 
     @property
-    def _build_subfolder(self):
-        return "build_subfolder"
-
-    def source(self):
-        tools.get(**self.conan_data["sources"][self.version])
-        extracted_dir = "ChaiScript-" + self.version
-        os.rename(extracted_dir, self._source_subfolder)
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "7",
+            "clang": "7",
+            "apple-clang": "10",
```

Comment:
Sorry I didn't catch this earlier, seems like the msvc compiler is also missing here:

```suggestion
            "apple-clang": "10",
            "Visual Studio": "15.0",
            "msvc": "191",
```

Reply:
I believe the `msvc` and `Visual Studio` entries in this dictionary may not be evaluated since the check only happens conditionally on `not msvc(self)` on line 56. @RubenRBS, may be a good opportunity to look into updating the template, since other recipes are currently applying a newer pattern for this.

Either way not a blocker for this PR.